core/prodtest_emu.py

```
@@ -0,0 +1,35 @@
+parser.add_argument("-m", "--model", default=os.environ.get("TREZOR_MODEL"))
+args, remaining = parser.parse_known_args()
+
+if not args.model:
+    print("No model specified. Use -m <model> or set TREZOR_MODEL.", file=sys.stderr)
+    sys.exit(1)
+
+with tempfile.TemporaryDirectory(prefix="prodtest_emu_") as profile_dir:
+    emulator = get_prodtest_emulator(model=args.model, profile_dir=profile_dir)
+    with emulator:
+        emulator.start()
+        process = subprocess.Popen(remaining)
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)